README.md

```diff
@@ -26,6 +26,16 @@ Simply install with:
 
     pip install topydo
 
+### Required dependencies
+
+* [arrow][8]          : Used to turn dates into a human readable version.
+* [backports.shutil_get_terminal_size][9] : Used to determine your terminal
+                                            window size. This function was
+                                            added to the standard library in
+                                            Python 3.3 and so is only
+                                            required in older version of
```

Comment:
Fixed in [c8631a](https://github.com/minchinweb/topydo/commit/c8631a6a5ec8477ea8e91295f9405e42e183a952)